core_acc_modules/DE_helper.py

```diff
@@ -0,0 +1,107 @@
+"""
+Author: Alexandra Lee
+Date Created: 18 December 2020
```

Comment:
Is there a reason that this is backdated two months? Is it from a different repo, or has it just been in the works for awhile?

Reply:
This was originally written in another repo. I'm not sure what the convention is for dates. Do you think it makes more sense to note where it was originally from and keep the date as is?